File: WORKSPACE(STS)/01Day_test_1/src/main/java/com/spring/Person.java
package com.spring;

public class Person {
	String name="홍길동";
	int age=20;
	public Person() {
		super();
		System.out.println("생성자 호출");
	}
	@Override
	public String toString() {
		return "Person [name=" + name + ", age=" + age + "]";
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	
	
}

File: WORKSPACE(typescript)/02_type Assertion/person.ts
export class Person{
  constructor(public name:string,
  public age:number,
  public address:string){ //생성자 매개변수 public:멤버변수로 사용, 할당
    console.log("생성자 호출");
  } 
  getName(){
    return this.name;
  }
  getAge(){
    return this.age;
  }
  getAddress(){
    return this.address;
  }

  

}